tests/test_util.py

```diff
@@ -26,3 +26,19 @@ def test_split_and_preprocess(read_data):
     assert len(y_train) <= TRAIN_SIZE * len(iris_data.data)
     assert (len(X_train) == len(y_train))
     assert (len(X_test) == len(y_test))
+
+
+def test_hyperopt2skopt_space():
+    s1 = {'a': hp.uniform('a', 0, 10)}
+    s1 = modelgym.util.hyperopt2skopt_space(s1)
+    s2 = Real(0, 10)
```

Comment:
wouldn't it be a bit nicer to have 2 lists of spaces: hyperopt_space = [{}, {}], skopt_space=[,,]
that you zip together by zip(hyperopt_space, skopt_space) and then iterate over result and assert that 1st item after hyperopt2skopt_space transform equals to the second item?